src/main/java/com/github/maxopoly/MemeMana/model/MemeManaOwner.java

```diff
@@ -1,7 +1,17 @@
 package com.github.maxopoly.MemeMana.model;
 
+import java.util.Set;
+
 public interface MemeManaOwner {
```

Comment:
I kept this interface as clean as possible so far, because as mentioned before, it's also supposed to represent groups who own mana. Those wont have any associations, alts etc.. Your changes to this class also tie to the problem with the tables for tracking daily mana gain/owner. This interface should just provide what is needed to do transactions between entities who own mana (be it a group/player/whatever) and not include any alt stuff.

Reply:
We'll see if this interface is even needed once AltManager/BanStick changes happen. My first instinct is not to use an interface for a sum type because that makes the sum open, but maybe that's OK.